test/fetch-mock-helper.js

```diff
@@ -0,0 +1,12 @@
+export function fixtureWithHeaders(fixture) {
+  return {
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: fixture
+  };
+}
+
+export default function fetchMockPostOnce(fetchMock, apiUrl, fixture) {
+  fetchMock.postOnce(apiUrl, fixtureWithHeaders(fixture));
+}
```

Comment:
I'm curious as to what this wrapper is helpful for (I may be missing some context).  Is this required due to bumping the graph package, or is it just a syntactically preferred way to write these tests?

Reply:
Discussed offline :)